package.json

```diff
@@ -63,19 +63,18 @@
     "@18f/identity-stylelint-config": "^2.0.0",
     "ansi-colors": "^4.1.3",
     "axe-core": "^4.10.0",
-    "browserify": "^17.0.0",
+    "browserify": "^17.0.1",
     "chalk": "^4.1.2",
     "cheerio": "^1.0.0",
     "chrome-launcher": "^0.14.1",
     "chrome-remote-interface": "^0.33.0",
     "concurrently": "^8.2.2",
     "del": "^6.1.1",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-import": "^2.30.0",
-    "express": "^4.21.0",
+    "eslint-plugin-import": "^2.31.0",
     "fancy-log": "^2.0.0",
-    "gulp": "^4.0.2",
-    "gulp-cli": "^2.3.0",
+    "gulp": "^5.0.0",
```

Comment:
No perceived changes from my initial testing but let me give this another sweep to make sure 👍

Reply:
Looks like there was an error with some blog post images being encoding in the `copyDocImages` function!

Resolved in [fdc8882](https://github.com/uswds/uswds-site/pull/2867/commits/fdc8882ec24b995d66cabedd2e4c12b3984917bd) and I haven't found any additional `gulp.src()` functions that need to be updated.

Builds currently failing due to Node LTS updgrade

Reply:
@mahoneycm build issues should be resolved now, but we have a new snyk error being flagged.

Reply:
@mejiaj  thanks for flagging! I looked into seeing if we can resolve the issue in compile. There is already a [compile issue](https://github.com/uswds/uswds-compile/pull/121) automatically created by Snyk but it's a breaking change. Del v7 and up requires ESM syntax.

Added the issue back to our snyk ignore for now but perhaps we should consider updating to ESM in the future?

Reply:
**question (non-blocking):** I don't think this needs to block this PR, but I wonder if the gulp dependency is needed here since the same version is included in uswds compile.